src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/None.xaml

```diff
@@ -2,99 +2,40 @@
 <!--Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.-->
 <Rule
   Name="None"
-  DisplayName="General"
   PageTemplate="generic"
-  Description="File Properties"
   PropertyPagesHidden="true"
   xmlns="http://schemas.microsoft.com/build/2009/properties">
-  <Rule.DataSource>
-    <DataSource Persistence="ProjectFile" HasConfigurationCondition="False" ItemType="None" SourceOfDefaultValue="AfterContext" />
-  </Rule.DataSource>
-  <Rule.Categories>
-    <Category Name="Advanced" DisplayName="Advanced" />
-    <Category Name="Misc" DisplayName="Misc" />
-  </Rule.Categories>
+    <Rule.DataSource>
+        <DataSource Persistence="ProjectFile" HasConfigurationCondition="False" ItemType="None" SourceOfDefaultValue="AfterContext" />
+    </Rule.DataSource>
+    <Rule.Categories>
+        <Category Name="Advanced" />
```

Comment:
Categories can also be removed from the "file" context. We no longer/never show the results of this file in any UI.